features/theme-mod-list.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Manage WordPress theme mods list
```

Comment:
The tests should be written with a list of theme mods that actually contain values, otherwise the tests could just succeed by random chance.

Reply:
You can use something like this to pre-populate the list:

```
Background:
  Given a WP install
  And I run `wp theme mod set key_a value_a`
  And I run `wp theme mod set key_b value_b`
```

Reply:
@schlessera Done. can you please check again? :smiley: